account_invoice_pricelist/tests/test_account_move_pricelist.py

```diff
@@ -27,6 +27,12 @@ def setUpClass(cls):
         cls.journal_sale = cls.env["account.journal"].create(
             {"name": "Test sale journal", "type": "sale", "code": "TEST_SJ"}
         )
+        # Make sure the currency of the company is USD, as this not always happens
```

Comment:
If I understand correctly your point, regarding V17.

```suggestion
        # Make sure the currency of the company is USD, as this not always happens
        # Remove this SQL request when porting this module in V17.
        # See : https://github.com/odoo/odoo/pull/107113 
```

Reply:
Added, @legalsylvain